x-pack/test/api_integration/apis/lens/existing_fields.ts

```diff
@@ -150,7 +151,7 @@ export default ({ getService }: FtrProviderContext) => {
           .get(
             `/api/lens/existing_fields/${encodeURIComponent(
               'logstash-*'
-            )}?fromDate=${TEST_START_TIME}&toDate=${TEST_END_TIME}`
+            )}?fromDate=${TEST_START_TIME}&toDate=${TEST_END_TIME}&dslQuery=${DSL_QUERY}`
```

Comment:
Nitpick: dslQuery is now required, but could we provide a default `match_all` query? But since this is only used by our client, this is probably fine.